loom/script/compiler/lsParser.cpp

```diff
@@ -2572,6 +2572,9 @@ Statement *Parser::parseEnumStatement()
     cls->metaTags = curTags;
     curTags.clear();
 
+    cls->docString = currentMultilineComment;
+    currentMultilineComment = "";
```

Comment:
This is a pattern that's used all over this code. It looks like it's a kind of a "used" flag. Could have been done cleaner with .clear() or something though, but then we'd probably want to make everything consistent.